database/migrations/2019_12_03_105400_create_action_stats_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateActionStatsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('action_stats', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('action_id')->index()->comment('The action id the stat is for.');
+            $table->foreign('action_id')->references('id')->on('actions');
+            $table->string('school_id', 255)->index()->comment('The school id the stat is for.');
+            $table->unsignedInteger('accepted_quantity')->comment('The sum of accepted post quantity for the school and action.');
```

Comment:
Ah sorry, one more nitpick! Can we name this `accepted_count` so it matches up with [the Campaign equivalent](https://github.com/DoSomething/rogue/blob/dfa30cb5e750826478ba3054a35c9a85fa187573/app/Http/Transformers/CampaignTransformer.php#L26)?

Reply:
This isn't the count of accepted posts though... it's their sum quantity (or "aggregate impact", in biz-cat speak)

Reply:
Oh you're totally right! I got myself mixed up there.